weapons.yaml

```diff
@@ -1054,65 +910,29 @@ HoverMissile:
 	Warhead@2Eff: CreateEffect
 		Explosions: small_grey_explosion
 		ImpactSounds: gexpifva.wav
-		InvalidImpactTypes: Water, Air
+		InvalidImpactTypes: Water
 	Warhead@3EffWater: CreateEffect
 		Explosions: small_watersplash
 		ImpactSounds: gexpwasa.wav
 		ValidImpactTypes: Water
-	Warhead@4EffAir: CreateEffect
-		Explosions: small_grey_explosion
-		ImpactSounds: gexpifva.wav
-		ValidImpactTypes: Air
 	Warhead@5: LeaveSmudge
 		SmudgeType: SmallCrater
 
 HoverMissileE:
-	ReloadDelay: 50
+	Inherits: HoverMissile
 	Burst: 4
-	Range: 6c0
 	Report: vifvatta.wav
 	ValidTargets: Ground, Air
-	Projectile: Missile
-		Speed: 213
-		Arm: 2
-		Blockable: false
-		Shadow: true
-		Inaccuracy: 128
-		Image: DRAGON
-		Palette: ra
-		HorizontalRateOfTurn: 8
-		RangeLimit: 35
-		ContrailColor: D8D8FF
-		ContrailLength: 16
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 80
-		ValidTargets: Ground, Air
-		Versus:
-			None: 100
-			Flak: 90
-			Plate: 80
-			Light: 70
-			Medium: 35
-			Heavy: 35
-			Wood: 75
-			Steel: 40
-			Concrete: 20
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: medium_grey_explosion
 		ImpactSounds: gexp15a.wav
 		InvalidImpactTypes: Water, Air
-	Warhead@3EffWater: CreateEffect
-		Explosions: small_watersplash
-		ImpactSounds: gexpwasa.wav
-		ValidImpactTypes: Water
 	Warhead@4EffAir: CreateEffect
```

Comment:
You can remove this entire warhead when you remove `Air` from `InvalidImpactTypes` of the second warhead.